firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -254,7 +254,6 @@ class ActivityStreamTest: BaseTestCase {
     func testContextMenuInLandscape() {
         // For iPhone test is failing to find top sites in landscape
         // can't scroll only to that area. Needs investigation
-        if iPad() {
```

Comment:
As per comment, let's leave the condition since the test has to pass in iPad devices too

Reply:
I ran the test in different iPhone and iPad simulators locally. All passed. But, yes, i am not aware of the iPad devices where this might be failing. So, reverting the change for now. Thanks for the context 🙏🏽